dnsapi/dns_dnsever.sh

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env sh
+
+#Here is a sample custom api script.
+#This file name is "dns_dnsever.sh"
+#So, here must be a method   dns_dnsever_add()
+#Which will be called by acme.sh to add the txt record to your api system.
+#returns 0 means success, otherwise error.
+#
+#Author: hiska
+#Report Bugs here: https://github.com/hiskang/acme.sh
+#
+########  Public functions #####################
+
+#Usage: dns_myapi_add   _acme-challenge.www.domain.com   "XKrxpRBosdIKFzxW_CT3KLZNf6q0HG9i01zxXp5CPBs"
+dns_dnsever_add() {
+  fulldomain=$1
+  txtvalue=$2
+  _info "Using dnsever"
+  _debug fulldomain "$fulldomain"
+  _debug txtvalue "$txtvalue"
+
+  if [ -z "$DNSEVER_ID" ] || [ -z "$DNSEVER_PW" ]; then
+    DNSEVER_ID=""
+    DNSEVER_PW=""
+    _err "You don't specify dnsever.com ID or PW yet."
+    _err "Please create you key and try again."
+    return 1
+  fi
+
+  #save the api key and email to the account conf file.
+  _saveaccountconf DNSEVER_ID "$DNSEVER_ID"
+  _saveaccountconf DNSEVER_PW "$DNSEVER_PW"
+
+  dnsever_txt "add" "$DNSEVER_ID" "$DNSEVER_PW" "$fulldomain" "$txtvalue"
+  return $?
+}
+
+#Usage: fulldomain txtvalue
+#Remove the txt record after validation.
+dns_dnsever_rm() {
+  fulldomain=$1
+  txtvalue=$2
+  _info "Using dnsever"
+  _debug fulldomain "$fulldomain"
+  _debug txtvalue "$txtvalue"
+
+  if [ -z "$DNSEVER_ID" ] || [ -z "$DNSEVER_PW" ]; then
+    DNSEVER_ID=""
+    DNSEVER_PW=""
+    _err "You don't specify dnsever.com ID or PW yet."
+    _err "Please create you key and try again."
+    return 1
+  fi
+
+  #save the api key and email to the account conf file.
+  _saveaccountconf DNSEVER_ID "$DNSEVER_ID"
+  _saveaccountconf DNSEVER_PW "$DNSEVER_PW"
+
+  dnsever_txt "delete" "$DNSEVER_ID" "$DNSEVER_PW" "$fulldomain" "$txtvalue"
+  return $?
+}
+
+####################  Private functions below ##################################
+
+dnsever_txt(){
+  action="$1"; login_id="$2"; login_password="$3"; fulldomain="$4"; txt="$5"
+
+  _inithttp
+
+  if [ "$_ACME_CURL" ] && [ "${ACME_USE_WGET:-0}" = "0" ]; then
```

Comment:
it's too details to do that,  here is how to deal with cookies.
https://github.com/Neilpang/acme.sh/blob/master/dnsapi/dns_cyon.sh#L159

That's to get the cookies headers, and set it back.

You should not know too much about wget or curl.